cassandra2/pom.xml

```diff
@@ -62,6 +62,14 @@ LICENSE file.
       <version>4.12</version>
       <scope>test</scope>
     </dependency>
+    <!-- only for Cassandra test (Cassandra 2.2+ uses Sigar for collecting system information, and Sigar requires some native lib files) -->
+	<dependency>
+	    <groupId>org.hyperic</groupId>
+	    <artifactId>sigar-dist</artifactId>
+	    <version>1.6.4.129</version>
+	    <type>zip</type>
+	    <scope>process-test-resources,test</scope>
```

Comment:
I didn't think this is a valid scope (ref [maven docs on dependency scopes](https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#Dependency_Scope)). I believe this should just be `test` scope.